test/run_envoy_bazel_coverage.sh

```diff
@@ -106,6 +110,15 @@ else
     cp bazel-out/_coverage/_coverage_report.dat "${COVERAGE_DATA}"
 fi
 
+# This worksaround https://github.com/bazelbuild/bazel/issues/19237
+echo "remove external files from coverage report"
+lcov -r "${COVERAGE_DATA}" "*external*" -o "${COVERAGE_DATA}.new"
```

Comment:
```suggestion
lcov -r "${COVERAGE_DATA}" "/external*" -o "${COVERAGE_DATA}.new"
```

this is removing mobile files it should not

Reply:
hack not required https://github.com/bazelbuild/bazel/issues/19237#issuecomment-1675999124